controller/api_test.go

```diff
@@ -120,6 +120,19 @@ func TestFunctionApi(t *testing.T) {
 	assert(len(funcs) == 2,
 		"created two functions, but didn't find them")
 
+	funcs_url := g.client.Url + "/v1/functions"
+	resp, err := http.Get(funcs_url)
+	panicIf(err)
+	assert(resp.StatusCode == 200, "http get status code on /v1/functions")
```

Comment:
Thanks for adding this test! You need a `defer resp.Body.Close()` somewhere here (it's only a test so it doesn't really matter, but still, good to add it).